chemprop/cli/train.py

```diff
@@ -352,31 +352,31 @@ def add_train_args(parser: ArgumentParser) -> ArgumentParser:
         "--task-weights",
         nargs="+",
         type=float,
-        help="the weight to apply to an individual task in the overall loss",
+        help="Weights to apply to an individual task in the overall loss",
```

Comment:
```suggestion
        help="Weights to apply for whole tasks in the loss function",
```
I feel this should reference `tasks` instead of `task` as you need to do something like `--task-weights 1, 1, 5`, i.e. suppling weights for every task. Not sure my wording is the best.